src/Plugins/Nop.Plugin.Product.Backup/Services/Export/ExportService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
+using Nop.Plugin.Product.Backup.Factory;
+using Nop.Plugin.Product.Backup.Services.Helpers;
+
+namespace Nop.Plugin.Product.Backup.Services.Export;
+
+public class ExportService : IExportService
+{
+    private readonly IProductBackupFactory _productBackupFactory;
+    private readonly IFileHelper _fileHelper;
+
+    public ExportService(IProductBackupFactory productBackupFactory, IFileHelper fileHelper)
+    {
+        _productBackupFactory = productBackupFactory;
+        _fileHelper = fileHelper;
+    }
+
+    public async Task ExportModel()
```

Comment:
name for this method is wrong, this is more like "Export" without model, i dont care what you have inside, the function/action is Export